boms/cloud-lts-bom/pom.xml

```diff
@@ -416,6 +417,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-logging-bom</artifactId>
```

Comment:
This includes the API surface. No irrelevant artifacts.
https://search.maven.org/artifact/com.google.cloud/google-cloud-logging-bom/3.4.0/pom

Reply:
Thanks for your clarifications.